Source/Widgets/Cesium3DTilesInspector/Cesium3DTilesInspector.js

```diff
@@ -201,12 +202,26 @@ define([
         tileDebugLabelsPanelContents.appendChild(makeCheckbox('showRenderingStatistics', 'Rendering Statistics'));
         tileDebugLabelsPanelContents.appendChild(makeCheckbox('showMemoryUsage', 'Memory Usage (MB)'));
 
+        optimizationPanelContents.appendChild(makeCheckbox('skipLevelOfDetail', 'Skip Tile LODs'));
+        var optSSEContainer = document.createElement('div');
+        optSSEContainer.appendChild(makeRangeInput('skipScreenSpaceErrorFactor', 1, 50, 1, 'Skip Screen Space Error Factor'));
```

Comment:
This should be called `skipScreenSpaceErrorFactorContainer`.

Also it's fine to remove the `opt` prefix from these.